packages/patrol_cli/lib/src/ios/ios_test_backend.dart

```diff
@@ -110,6 +110,7 @@ class IOSTestBackend {
         options.buildForTestingInvocation(),
         runInShell: true,
         workingDirectory: _fs.currentDirectory.childDirectory('ios').path,
+        environment: Map.castFrom(options.flutter.dartDefineFromFile),
```

Comment:
I'm pretty sure specifying `environment` here (and below as well) is not needed.

The dart defines are passed to `flutter build ios --config-only` invocation above, and it handles base64-encoding them in `ios/Flutter/Generated.xcconfig`.